### DISCUSSION TITLE
Font style not changing

### DISCUSSION THREAD

**Discussion by WIZN0DE** [inci-august/august-themes#2] [Q&A] [answered]
Hi, I just installed august themes and applied the drawbridge theme but the font style doesn't seem to change from the default one. I tried uninstalling and reinstalling it but couldn't get it to work. Can someone please help me with this issue.

**Comment by inci-august** [comment, marked as answer]
Hi, @SatsuiN0Hado. Themes usually don't set a font. You can customize it yourself by going into <kbd>Settings</kbd>-><kbd>Text Editor</kbd>-><kbd>Font</kbd>. By the way, make sure you have the font you want installed on your computer :)

**Reply by WIZN0DE** [reply]
Thank you @inci-august I was able to change the font. Love your themes🥰

**Comment by chaitanyabsprip** [comment]
@inci-august what font is being used in the screenshots?
